### PR TITLE
SAK-46280 Forums : fix encoding for mailtos

### PR DESCRIPTION
**Pull request by bgarciaentornos**
Not possible to solve using the existing component EscapedOutputLinkRenderer (there's another encoding afterwards), so removing it.

Adding missing i18n too

### CODE REVIEW

File: msgcntr/messageforums-app/src/webapp/js/forum.js [hunk 1]
```diff
@@ -850,3 +850,10 @@ MFR_RBC.saveRubric = function() {
     const rubricGrading = document.getElementsByTagName("sakai-rubric-grading").item(0);
     rubricGrading && rubricGrading.release();
 };
+
+window.onload = function() {
+    var mailtos = document.querySelectorAll("[id*=createEmail1");
+    mailtos.forEach(item =>{
+        item.setAttribute('href', item.href.replaceAll('+',' '));
+    });
```
Comment (adrianfish) [inline msgcntr/messageforums-app/src/webapp/js/forum.js:858]:
```suggestion
    document.querySelectorAll("[id*=createEmail1").forEach(item => {
        item.setAttribute('href', item.href.replaceAll('+', ' '));
    });
```

Comment (adrianfish) [reply msgcntr/messageforums-app/src/webapp/js/forum.js:858]: Actually, shouldn't you just use setAttribute and getAttribute, or do you want to do the replace on the absolute url?

**Review by adrianfish — CHANGES_REQUESTED** [review]

**Review by ern — CHANGES_REQUESTED** [review]
Don't you think it would be better to fix the issue that is  creating the markup incorrectly?

**Comment by bgarciaentornos** [comment]
That's how it was done previously with the EscapedOutputLinkRenderer class, but now with this jsf version I couldn't find a way.

**Review by mpellicer — APPROVED** [review]

**Pull request merged**